Dockerfiles/manifests/agent-only/daemonset.yaml

```diff
@@ -2,137 +2,145 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: datadog-agent
+  name: datadog
```

Comment:
😔 FYI when you regenerated these files and change the `name` field, you break the resource's compatibility with past deployments because you've changed it's unique identifier.

Now I've got `datadog` and `datadog-agent` floating around and have to reconcile these objects.